src/d3a/setup/redis_communication_default_2a.py

```diff
@@ -47,17 +49,16 @@ def get_setup(config):
                 'House 2',
                 [
                     Area('H2 General Load', strategy=LoadHoursStrategy(avg_power_W=200,
-                                                                       hrs_per_day=4,
+                                                                       hrs_per_day=24,
                                                                        hrs_of_day=list(
-                                                                           range(12, 16)),
+                                                                           range(0, 24)),
                                                                        final_buying_rate=35),
                          appliance=SwitchableAppliance()),
                     Area('H2 PV', strategy=PVStrategy(4, 80),
                          appliance=PVAppliance()),
 
                 ],
                 transfer_fee_pct=0, transfer_fee_const=0,
-
```

Comment:
Any particular reason to change parameters in this setup file?

Reply:
No, Just to have loads trying to buy at the first market slot, not on midday.

Reply:
@hannesdiedrich any reason for that? Otherwise I will revert this setup file to its original version.

Reply:
@spyrostz please see my comment above.

Reply:
@hannesdiedrich sorry, just saw your comment. I will revert this file to its original state if it is fine for you then.